test.sh

```diff
@@ -135,8 +135,6 @@ gometalinter.v2 --disable-all \
     --enable=interfacer \
     --enable=misspell \
     --enable=gocyclo \
-    --line-length=170 \
```

Comment:
Why removing this?

Reply:
Because I went over the line length in a template :P